lib/ufe/UsdObject3d.cpp

```diff
@@ -9,17 +9,38 @@
 #include "UsdObject3d.h"
 #include "Utils.h"
 
+#include "ufe/attributes.h"
 #include "ufe/types.h"
 
-#include "pxr/usd/usdGeom/bboxCache.h"
-#include "pxr/usd/usd/timeCode.h"
+#include <pxr/usd/usdGeom/bboxCache.h>
+#include <pxr/usd/usd/timeCode.h>
+#include <pxr/usd/usdGeom/tokens.h>
+
+#include <stdexcept>
 
 namespace {
 Ufe::Vector3d toVector3d(const GfVec3d& v)
 {
     return Ufe::Vector3d(v[0], v[1], v[2]);
 }
 
+Ufe::AttributeEnumString::Ptr getVisibilityAttribute(Ufe::SceneItem::Ptr item)
```

Comment:
I'm getting an "unused-function" error here (compiling on strict mode). Maybe I'm missing something since everybody else seems to be able to build it...

Reply:
Thanks for catching this. I've updated the code by #if protecting this function. It's only used below by other protected code so the warning/error you got is valid.

Reply:
It compiles now. Thanks!

Reply:
Thanks Pilar, that's exactly the reason why we appreciate your external review.